src/main/webapp/app/utils/navigation.utils.ts

```diff
@@ -9,6 +9,18 @@ export const navigateBack = (router: Router, fallbackUrl: string[]): void => {
     }
 };
 
+export const navigateToExampleSubmissions = (router: Router, exercise: Exercise): void => {
+    setTimeout(() => {
+        router.navigate(['course-management', exercise.course!.id!, exercise.type! + '-exercises', exercise.id, 'example-submissions']);
```

Comment:
Don't we need to account for exam exercises here? I.e. if `exercise.exerciseGroup` is set (see the method below).

Reply:
thanks for reminding me @FrankeLukas
Done.